job-service-db/src/main/resources/v2.7.0/changelog.xml

```diff
@@ -149,12 +149,14 @@
         </addColumn>
     </changeSet>
     <changeSet id="add_procedure_internal_get_options" author="Gamma Team">
+        <validCheckSum>7:bc95dd9476c92f7dcc586d445bf0dbf4</validCheckSum>
```

Comment:
@dermot-hardy Is this ok to be changing? No idea why these 2 sql files checksum has changed and the others didn't when the copyright was updated.

Reply:
@Anthony-McGreevy Why are these changing but others don't need to change?

Reply:
These are the only two whose checksums have included the license header in the 2.7 release. Now that we have changed that file their hashes need to be updated so that we can still upgrade from that releases version of the database.

Reply:
@dermot-hardy @michael-bryson I tried changing the header again to 2021 as Michael suggested and again it resulted in a checksum failure to validate. So I'd say we can be confident that the copyright header is being included in the checksum for these files. Don't think this will be much of an issue in the future as it can just be updated as part of the change to the copyright headers. The build actually states "the checksum is now:" and then gives you the new checksum when it does not match what it was expecting, so updating in future will not be a big undertaking. I also tried upgrading against the 3.1 version of the database just in case there was some issue with jumping a version but the result was the same.